### DISCUSSION TITLE
(Feature request) Sharing over a different Port

### DISCUSSION THREAD

**Discussion by CrO2Cl2** [TriliumNext/Trilium#7341] [Q&A] [answered]
### Describe feature

I run trilium through a VPN for added security, but would still like to have the option to share notes on the open internet. The ability to pick a port that will be used for sharing only would allow me to route the sharing of the notes through my reverse proxy and into the open internet.

**Comment by ReinforceZwei** [comment, marked as answer]
You could configurate your reverse proxy to only route your shared notes if the requested path is under `/share`.

For example caddy server
```caddyfile
http://domain.com {
  reverse_proxy /share http://localhost:8080/share
  reverse_proxy /assets http://localhost:8080/assets
}
```
